clcache.py

```diff
@@ -1334,7 +1334,7 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
```

Comment:
Right, so the question remains: is `origCmdLine` a backup or an alias. I think the first one is more obviours and we should do `origCmdLine = cmdLine.copy()` no matter how we deal with the place below.

btw. copy() is not a deep copy (as I thought before), but a shallow copy, so it only copies a bunch of pointers to the (immutable) string elements.